src/workers/notifications/index.ts

```diff
@@ -0,0 +1,55 @@
+import { NotificationWorker } from './worker';
+import { Worker } from '../worker';
+import {
+  generateNotification,
+  storeNotificationBundle,
+} from '../../notifications';
+import communityPicksFailed from './communityPicksFailed';
+import communityPicksSucceeded from './communityPicksSucceeded';
+import communityPicksGranted from './communityPicksGranted';
+import articlePicked from './articlePicked';
+import articleNewCommentPostCommented from './articleNewCommentPostCommented';
+import articleNewCommentCommentCommented from './articleNewCommentCommentCommented';
+import articleUpvoteMilestone from './articleUpvoteMilestone';
+import articleReportApproved from './articleReportApproved';
+import articleAnalytics from './articleAnalytics';
+import sourceRequest from './sourceRequest';
+import commentMention from './commentMention';
+import commentReply from './commentReply';
+import commentUpvoteMilestone from './commentUpvoteMilestone';
+
+function notificationWorkerToWorker(worker: NotificationWorker): Worker {
+  return {
+    ...worker,
+    handler: async (message, con, logger) => {
+      const args = await worker.handler(message, con, logger);
+      if (!args) {
+        return;
+      }
+      const bundles = args.map(({ type, ctx }) =>
+        generateNotification(type, ctx),
```

Comment:
The ctx is always singular right?

Reply:
What do you mean?

Reply:
Regarding this, It showed clearly down the line of my review, forgot to remove this.